sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -258,11 +254,10 @@ Run a predictive model to identify a collection of named entities in the passed-
 ```C# Snippet:RecognizeEntitiesAsync
 string input = "Microsoft was founded by Bill Gates and Paul Allen.";
 
-Response<IReadOnlyCollection<CategorizedEntity>> response = await client.RecognizeEntitiesAsync(input);
-IEnumerable<CategorizedEntity> entities = response.Value;
+Response<IReadOnlyCollection<CategorizedEntity>> entities = await client.RecognizeEntitiesAsync(input);
```

Comment:
Nitpick, but above in linkedentities you make it equal to the .value of the response, so later on you don't need the .Value

Reply:
Because this is an async method, I would have to do something like `IReadOnlyCollection<CategorizedEntity> entities = client.RecognizeEntitiesAsync(input).ConfigureAwait(false).GetAwaiter().GetResult().Value`
which doesn't look that good. So leaving as is